.gitignore

```diff
@@ -20,3 +20,11 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+.pnp.*
+.yarn/*
+!.yarn/patches
+!.yarn/plugins
+!.yarn/releases
```

Comment:
We're using Corepack, we don't need to commit Yarn. Can you also `git rm .yarn/releases/yarn-3.2.3.cjs` please?
```suggestion
```